CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## [7.3.1](https://github.com/commercetools/merchant-center-application-kit/compare/v7.3.0...v7.3.1) (2019-02-04)
+
+#### 🐛 Type: Bug
+
+- `application-shell`
+  - [#341](https://github.com/commercetools/merchant-center-application-kit/pull/341) fix(apollo-cache): to use \_\_typename + key for whitelisted types only if the id is not present ([@emmenko](https://github.com/emmenko))
+
+#### 🚀 Type: New Feature
```

Comment:
I thought if this was big enough of a feat but if it is one it's 7.4.0? I don't mind too much.

Reply:
I guess...I wouldn't consider it like a real feature but whatever, I don't mind much